init.lua

```diff
@@ -200,6 +213,8 @@ function CheckOptions(data, entity, distance)
 	if distance and data.distance and distance > data.distance then return false end
 	if data.job and not JobCheck(data.job) then return false end
 	if data.excludejob and JobCheck(data.excludejob) then return false end
+	if data.jobType and not JobTypeCheck(data.jobType) then return false end
+	if data.excludejobType and not JobTypeCheck(data.excludejobType) then return false end
```

Comment:
I suspect the logic for this should be checking if we **are** an exluded jobType rather than **not**:
```lua
if data.excludejobType and JobTypeCheck(data.excludejobType) then return false end
```
(removed the `not`)